### DISCUSSION TITLE
Tiktok integration

### DISCUSSION THREAD

**Discussion by CGJoe-code** [baffler/Transparent-Twitch-Chat-Overlay#348] [Ideas]
I would love for a tool to be able to combine chats into one transparent tool so its useful for those with one monitor.
Is this a possible feature?

**Comment by jwizfoshizz** [comment]
did you find solution

**Comment by Stefanzoka** [comment]
U can link all accounts on botrix. Configure chat on this site and then add new window in transparent twitch chat overlay and paste link to the botrix "all chat". Thats all, ez.
